src/test/java/edu/ucsb/cs156/frontiers/annotations/WithInstructorCoursePermissions.java

```diff
@@ -0,0 +1,11 @@
+package edu.ucsb.cs156.frontiers.annotations;
+
+import org.springframework.security.test.context.support.WithMockUser;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+

```

Comment:
I feel like this is the kind of code where some Javadoc would be particularly useful, i.e. something that explains what the annotation `@WithInstructorCoursePermissions` does, what the use case is for it, and what developers need to know about it.